packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -0,0 +1,199 @@
+    JurisdictionCardTagClass,
+} from './JurisdictionCard.classnames';
+import JurisdictionCardBack from './JurisdictionCardBack';
+import JurisdictionCardRow from './JurisdictionCardRow';
+import JurisdictionCardTag from './JurisdictionCardTag';
+import JurisdictionCardVerificationTag from './JurisdictionCardVerificationTag';
+
+type TJurisdictionCardProps = JurisdictionCardClassProps & {
+    jurisdiction: string;
+    onSelect: (clickedJurisdiction: string) => void;
+    tag?: string;
+};
+
+type TDisplayTextSkinColor =
+    | 'brown-dark'
+    | 'red-dark'
+    | 'red-darker'
+    | 'red-light'
+    | 'voilet-dark'
+    | 'yellow-dark'
+    | 'yellow-light';
+
+type TVerificationDocumentsMapper = {
+    [key: string]: {
+        category: 'poa' | 'poi' | null;
+        icon?: JSX.Element;
+    };
+};
+
+const verificationDocumentsMapper: TVerificationDocumentsMapper = {
+    documentNumber: {
+        category: 'poi',
+        icon: <IdCardIcon />,
+    },
+    nameAndAddress: {
+        category: 'poa',
+        icon: <DocumentsIcon />,
+    },
+    notApplicable: {
+        category: null,
+    },
+    selfie: {
+        category: 'poi',
+        icon: <SelfieIcon />,
+    },
+};
+
+const JurisdictionCard = ({
+    isAdded = false,
+    isSelected = false,
+    jurisdiction = 'svg',
+    onSelect,
+}: TJurisdictionCardProps) => {
+    const [isFlipped, setIsFlipped] = useState(false);
+    const { toggleDynamicLeverage } = useDynamicLeverageModalState();
+    const { getModalState } = useModal();
+
+    const descriptionClickHandler = (tag?: string) => (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
+        event.stopPropagation();
+        if (tag === 'dynamicLeverage') {
+            toggleDynamicLeverage();
+        } else {
+            setIsFlipped(true);
+        }
+    };
+
+    const { contents, header, isOverHeaderAvailable, overHeader, verificationDocs } = useMemo<TJurisdictionCardItems>(
+        () => getJurisdictionContents()[jurisdiction],
+        [jurisdiction]
+    );
+    const marketType = getModalState('marketType') || 'all';
+    const rows = contents[marketType] || [];
+
+    const parseDescription = (row: TJurisdictionCardSection) => {
+        if (row.clickableDescription)
+            return row.clickableDescription.map(description => {
+                if (description.type === 'link') {
+                    return (
+                        <a
+                            className='text-brand-red-light '
+                            key={description.text}
+                            onClick={descriptionClickHandler(description.tag)}
+                            onKeyDown={event => {
+                                if (event.key === 'Enter') {
+                                    descriptionClickHandler(description.tag);
+                                }
+                            }}
+                        >
+                            {description.text}
+                        </a>
```

Comment:
Don't we have a static link component form Tradershub components? If we do and it follows the design can we incorporate it here instead of writing it on our own